### PR TITLE
Adds a snack called "Old Empire Bar" as a reference to Age of Empires.

### PR DESCRIPTION
**Pull request by SysadminJeroen**
A few days ago the IC chat wasn't sanitized and I accidentally played `rogan.mp3`, I thought that it was a 1 second audio file but it was a 9 minute audio file of "Rogan?" repeating over and over.
Worked on this for the last two hours but I am finally at a stage where I am confident at making a pull request.

ss 1
ss 2
ss 3
ss 4
ss 5

### CODE REVIEW

**Comment by Imagebard** [comment]
This PR is an abomination in the eyes of the lord.

**Comment by SysadminJeroen** [comment]
@Imagebard Wololo

**Comment by TheFriggenSawbones** [comment]
Please add this

**Comment by SysadminJeroen** [comment]
@Imagebard Getting b& (for 3 days) is a quite good way to fight my SS13 addiction and to become productive.

**Comment by SkowronX** [comment]
2219fcc5506b0fa93b8b44db474a5541

**Comment by clusterfack** [comment]
Prostagma?

**Comment by SysadminJeroen** [comment]
@clusterfack Could add it as a notice or as a description.

**Comment by Iamgoofball** [comment]
@clusterfack 
yow

**Comment by Intigracy** [comment]
Tbh I don't think you should have been banned for that rogan MP3, that was funny as fuck.

**Comment by ihadtoregisterforthis** [comment]
:+1: bone it

**Comment by clusterfack** [comment]
Test

**Comment by SangriaMonkey** [comment]
:ok_hand:

**Comment by ghost** [comment]
Tower Rush time

The best offense is defense

**Comment by SysadminJeroen** [comment]
Add certain taunts like "Dagdum" or "All hail, King of the losers!" to the possible notices?
http://ageofempires.wikia.com/wiki/Taunts
Edit: Vote yes or no using the thumbs up and down.

**Comment by Iamgoofball** [comment]
SURE, blame it on your master controller

**Comment by Kitfox88** [comment]
i'm really sad i missed the rogan thing

**Comment by Intigracy** [comment]
I think you should change the rogan stuff.

Instead of it being said to you in chat, you should be forced to say it.

**Comment by SysadminJeroen** [comment]
@Intigracy Currently the only think you are forced to say is "Rogan?".

Everything else is a notice to you that only you can read.

**Comment by ihadtoregisterforthis** [comment]
The actual taunt says "Roggan", with two gees

**Comment by SysadminJeroen** [comment]
@ihadtoregisterforthis I know, but "Rogan" is widely used as well.

**Comment by SysadminJeroen** [comment]
Rebased, picked "ours" (which is confusing), put my sprite back in the sprite bank and pushed it.

I have also changed some of the notices around.

I'll test this again.

**Comment by SysadminJeroen** [comment]
Travis says the build passes, but I ain't buying that.

```
loading vgstation13.dme
loading interface\skin.dmf
code\modules\events\mob_swarm.dm:41:warning: mobber: variable defined but not used
loading tgstation.dmm
saving vgstation13.dmb (DEBUG mode)
vgstation13.dmb - 0 errors, 1 warning (4/3/16 5:08 pm)
```

**Comment by SysadminJeroen** [comment]
Looks like the issue exists in ffba5cdcc40bf1c573f7fc737136556594f9543d.

**Comment by Shadowmech88** [comment]
@Jeroen52 
When rebasing, "ours" refers to yours. "Theirs" is the main branch.
Ideally,  you'd use the conflict solver to manually view and resolve the conflicts. Resolving automatically occasionally means some new code might be replaced with your older code, if you've edited the files near where they're conflicting.

**Comment by SysadminJeroen** [comment]
@Shadowmech88
What I mean with "ours" I mean Git's definition of "ours" and "theirs".
The conflict was in a binary, so I couldn't use KDiff3 to resolve the issue.
What I did was just get all the changes from `Bleeding-Edge` and apply my changes on top of that, and do not let my changes overwrite those of `Bleeding-Edge` if there is a merge collision and use theirs automatically. After that I added my sprite back in the sprite bank.

**Comment by ihadtoregisterforthis** [comment]
Dylan touched something called like that on #9212

> A strange dirty edit in vgstation13.dme, also has anyone noticed that new compile warning ?

**Comment by SysadminJeroen** [comment]
@ihadtoregisterforthis, the warning exists since c9096aa4cb1226f2dd16bd8308c9777cb0d49578.

**Comment by SysadminJeroen** [comment]
cc: @gbasood

**Comment by SysadminJeroen** [comment]
Not behind my PC at home, and somehow can't add my repo SSH url as a
remote. Please [DNM] until I have added this patch when I'm home.

**Comment by SysadminJeroen** [comment]
Added the changelog.
Can remove the [Conflicts] tag as there is currently no conflict.
I'll rebase and pray that the compiling warning is no longer present.
Edit: Looks like the warning has not resolved yet, hopefully it won't impact the features.

**Comment by SysadminJeroen** [comment]
Everything looks like it works as intended.
You can remove the [Do Not Merge] label because I think that I got everything.

**Comment by ghost** [comment]
Pouring holy water on it changing it from blue to red, and vice versa when?

**Comment by ihadtoregisterforthis** [comment]
@Skullyton only if it makes a "Wololo" message.

**Comment by SysadminJeroen** [comment]
@Skullyton @ihadtoregisterforthis I am not good enough for that yet.

@PJB3005 No merge conflict as of the moment, can you remove the label?

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -168,6 +168,23 @@
...
+	reagent_state = LIQUID
+	color = "#0000FF"
+	custom_metabolism = 0.01
+
+/datum/reagent/rogan/on_mob_life(var/mob/living/M)
+	if(..()) return 1
+
+	if(prob(1))
+		if(prob(42))
+			to_chat(M, "<span class='notice'>[pick("Rogan?", "ROGAN.", "Food please.", "Wood please.", "Gold please.", "All hail, king of the losers!", "I'll beat you back to Age of Empires.", "Sure, blame it on your ISP.", "Start the game already!", "It is good to be the king.", "Long time, no siege.", "Nice town, I'll take it.", "Raiding party!", "Dadgum.", "Wololo.", "Attack an enemy now.", "Cease creating extra villagers.", "Create extra villagers.", "Build a navy.", "	Stop building a navy.", "Wait for my signal to attack.", "Build a wonder.", "Give me your extra resources.", "What age are you in?")]")
```
Comment (Iamgoofball) [inline code/modules/reagents/Chemistry-Reagents.dm:185]: gonna go play aoe II again thanks brb

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 2]
```diff
@@ -168,6 +168,23 @@
...
+		else
+			M.say("Rogan?")
```
Comment (Iamgoofball) [inline code/modules/reagents/Chemistry-Reagents.dm:187]: this should be the same list as above

Comment (SysadminJeroen) [inline code/modules/reagents/Chemistry-Reagents.dm:187]: @Iamgoofball I played rogan.mp3 using the exploit. So that is why the only word here is Rogan.

Comment (Intigracy) [inline code/modules/reagents/Chemistry-Reagents.dm:187]:
```
M.say("Rogan?")
spawn(20)
   M.say(Rogan?)
   spawn(20)
      M.say(Rogan?)
```

:^)

**Pull request merged**